test/integration/README.md

```diff
@@ -111,6 +111,19 @@ reused in other integration tests. If it's likely be reused, please add the
 appropriate functions to existing utilities or add new test utilities. If it's
 likely a one-off change, it can be scoped to the existing test file.
 
+# Debugging integration tests
+
+The Envoy integration test fraemwork is generally designed to fast-fail when
```

Comment:
s/fraemwork/framework. I'm guessing this is a test to see if anyone is reading the README's :smile: